platform/lifecycle_inputs_test.go

```diff
@@ -172,6 +175,7 @@ func testLifecycleInputs(t *testing.T, when spec.G, it spec.S) {
 				h.AssertEq(t, inputs.UID, 1234)
 				h.AssertEq(t, inputs.UseDaemon, true)
 				h.AssertEq(t, inputs.UseLayout, true)
+				h.AssertEq(t, inputs.InsecureRegistry, str.Slice(nil))
```

Comment:
Shouldn't this expect `some-insecure-registry`?

Reply:
We should probably also check that we can parse a comma separated list...

Reply:
Yeah, I tried to copy the UseExtendendFlags way but I got it wrong.
Thanks for pointing this out Natalie!